addons/website_forum/views/website_forum.xml

```diff
@@ -216,78 +216,83 @@
-            </a>
-            <span t-if="not question.active and question.state!='offensive'"><b> [Deleted]</b></span>
-            <span t-if="not question.active and question.state=='offensive'"><b> [Offensive]</b></span>
-            <span t-if="not question.active and question.state=='offensive' and question.closed_reason_id"><b> [<t t-esc="question.closed_reason_id.name[0].upper() + question.closed_reason_id.name[1:]"/>]</b></span>
-            <span t-if="question.state == 'close'"><b> [Closed]</b></span>
-            <t t-if="question.post_type == 'link'" >
-                <a title="Redirect to external link" aria-label="External link" t-att-href="question.content_link" class="text-muted"><i class="fa fa-external-link forum_link"/></a>
-            </t>
-            <span t-if="question.has_validated_answer" title="Question has accepted answer" role="img" aria-label="Question has accepted answer"><i class="fa fa-check-circle oe_answer_true"/></span>
-        </div>
-        <div><t t-raw="post_content"/></div>
-        <t t-foreach="question.tag_ids" t-as="question_tag">
-            <a t-attf-href="/forum/#{ slug(forum) }/tag/#{slug(question_tag)}/questions">
-                <span t-attf-class="float-right badge #{tag and tag.name == question_tag.name and 'badge-primary' or 'badge-secondary'}" t-field="question_tag.name"
-                    style="margin-right: 4px;"/>
-            </a>
-        </t>
-        <small class="text-muted">
-            <i t-if="question.post_type == 'question'" t-attf-class="fa fa-question-circle" role="img" aria-label="Question" t-att-title="question.post_type.capitalize()" />
-            <i t-if="question.post_type == 'discussion'" t-attf-class="fa fa-comments-o" role="img" aria-label="Discussion" t-att-title="question.post_type.capitalize()" />
-            <i t-if="question.post_type == 'link'" t-attf-class="fa fa-link" role="img" aria-label="Link" t-att-title="question.post_type.capitalize()" />
-            <span>By </span><span t-field="question.create_uid" t-options='{"widget": "contact", "country_image": True, "fields": ["name", "country_id"]}' style="display: inline-block;"/>
-            <span> • </span><span t-field="question.write_date" t-options='{"format":"short"}'/>
-            <span> • </span><span t-field="question.views" /> <t t-if="question.views&lt;=1">View</t><t t-else="">Views</t>
-            <span> • </span><a t-attf-href="/forum/#{ slug(forum) }/question/#{ slug(question) }">
-
-                <t t-esc="question.child_count"/>
-                <t t-if="question.post_type == 'question'">
-                    <t t-if="question.child_count&lt;=1">Answer</t>
-                    <t t-else="">Answers</t>
-                </t>
-                <t t-if="question.post_type != 'question'">
-                    <t t-if="question.child_count&lt;=1">Comment</t>
-                    <t t-else="">Comments</t>
+    <div t-if="question.has_validated_answer" class="validate_answer">
```

Comment:
Clearly this should not be there as its neighbors seems to be a col and this is not a col :sob: